plugins/workspace-minimap/src/focus_region.ts

```diff
@@ -187,9 +191,9 @@ export class FocusRegion {
     top += (minimapSvg.height - minimapContent.height) / 2;
 
     // Set the svg attributes.
-    this.rect.setAttribute('transform', `translate(${left},${top})`);
-    this.rect.setAttribute('width', width.toString());
-    this.rect.setAttribute('height', height.toString());
+    this.rect?.setAttribute('transform', `translate(${left},${top})`);
+    this.rect?.setAttribute('width', width.toString());
+    this.rect?.setAttribute('height', height.toString());
```

Comment:
Instead of using `?` can you throw an error at the top of this function if `this.rect` is not initialized?

Like:
```
if (!this.rect) {
  throw new Error('The focus region must be initialized (`init`) before calling `update`');
}
```